---
content_type: page
title: Lecture Notes
uid: d8caf4b5-65f9-9b6e-1c0f-342dc94199df
---

| SESÂ # | TOPICS |
| --- | --- |
| L0 | Introduction - _Prof. Leonid Mirny_ (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 1.8 MB]({{< baseurl >}}/resources/lecture0)) |
| {{< td-colspan 3 >}}**Module 1: Evolutionary and Population Genetics - _Lectures by Prof. Robert Berwick_**{{< /td-colspan >}} |||
| L1 | The Basic Forces of Evolution: Mutation, Recombination and Mating, Migration, Neutral Evolution and Drift, Effective Population Size ([PDF]({{< baseurl >}}/resources/hst508lecture1)) |
| L2 | Selection, Fitness, Probability of Fixation, Coalescent Theory (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 3.3 MB]({{< baseurl >}}/resources/hst2))  {{< br >}}  {{< br >}}Parameter Estimation |
| R1 | The Basic Forces of Evolution: Mutation, Recombination and Mating, Migration, Neutral Evolution and Drift, Effective Population Size  {{< br >}}  {{< br >}}Selection, Fitness, Probability of Fixation, Coalescent Theory  {{< br >}}  {{< br >}}Parameter Estimation |
| L3 | Selection, Fitness, and Diffusion Models (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 3.4 MB]({{< baseurl >}}/resources/hst3))  {{< br >}}  {{< br >}}Molecular Evolution, Jukes-Cantor Model, Kimura Model |
| L4 | Human/Mouse/Rat Examples  {{< br >}}  {{< br >}}Measuring Selection: From 'Classical' Methods to Maximum Likelihood (With Applications to Disease Evolution, HIV and Influenza)  {{< br >}}  {{< br >}}McDonald-Kreitman Test |
| R2 | Discussion of Problem Set 1 |
| L5 | Medical Lecture: Genetic Diversity and Evolution of Hepatitis C Virus - _Prof. Isaac Kohane_ |
| L6 | Guest Lecture - _Prof. Richard Lewontin_ |
| {{< td-colspan 3 >}}**Module 2: Comparative Genomics - _Lectures by Prof. Shamil Sunyaev_**{{< /td-colspan >}} |||
| L7 | Domain Structure of Proteins, Sequence-Structure-Function Relationships  {{< br >}}  {{< br >}}Orthologs, Paralogs, In/Out-Paralogs  {{< br >}}  {{< br >}}_Pairwise Sequence Comparison  {{< br >}}_\- Substitution Matrices (PAM, BLOSUM, Gonnet)  {{< br >}}\- Alignment Methods  {{< br >}}\- Alignment Statistics |
| L8 | Structure of the Genome  {{< br >}}  {{< br >}}Genome Comparison and Genome Evolution: Duplication, Recombination, Insertions, Repeats  {{< br >}}  {{< br >}}Finding Genome Elements: Genes, Regulatory Regions  {{< br >}}  {{< br >}}Motif Searching  {{< br >}}  {{< br >}}Conserved Non-coding |
| R3 | Infinite Site Model: Nucleotide Diversity, Allele Frequency |
| L9 | Multiple Alignments, Profiles and PSSM, HMMs, Domain Databases  {{< br >}}  {{< br >}}Prediction of Gene Function Using: Homology, Context, Structure |
| L10 | SNPs: Microevolution, Inference of Population History  {{< br >}}  {{< br >}}LD, Haplotypes, Medical Applications  {{< br >}}  {{< br >}}Effect of SNPs on Function |
| R4 | Recitation: SNPs |
| L11 | Medical Lecture: Human Variations Genes, Genotypes and Generations - _Dr. Marco Ramoni_ (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF]({{< baseurl >}}/resources/ra_hst508_103105)) (Courtesy of Dr. Marco Ramoni. Used with permission.) |
| {{< td-colspan 3 >}}**Module 3: Structural Genomics and Proteomics - _Lectures by Prof. Leonid Mirny_**{{< /td-colspan >}} |||
| L12 | Overview of Protein Structures, Domain Architecture (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 1.2 MB]({{< baseurl >}}/resources/lecture_st1))  {{< br >}}  {{< br >}}Sequence-structure Mapping, Protein Folding, Forces and Interactions |
| L13 | Structure-based Substitution Matrices (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 1.1 MB]({{< baseurl >}}/resources/lecture_st2))  {{< br >}}  {{< br >}}Protein Structure Prediction  {{< br >}}  {{< br >}}Threading  {{< br >}}  {{< br >}}Protein Function: Binding and Kinetics, Michaelis-Menthen Kinetics, Inhibition  {{< br >}}  {{< br >}}Protein-DNA Recognition: Models and Algorithms |
| L14 | Proteomics: Networks of Protein-protein Interactions, Complexes, Modules  {{< br >}}  {{< br >}}Power-law Distributions, Clustering Coefficient |
| L15 | Proteomics: Evolution of Protein and Regulatory Networks, Function and Misfunction of Genes in Networks |
| R5 | Recitation |
| L16 | Medical Lecture: Hemoglobin and the Anemias - _Prof. Isaac Kohane_ |
| L17 | Guest Lecture |
| {{< td-colspan 3 >}}**Module 4: Functional Genomics and Networks - _Lectures by Prof. Alvin Kho_**{{< /td-colspan >}} |||
| L18 | Gene Regulation and Function, Conservation, Detecting Regulatory Elements (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 2.0 MB]({{< baseurl >}}/resources/hst508_113005_l1)) |
| L19 | RNA Expression: Clustering and Classification (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF - 5.0 MB]({{< baseurl >}}/resources/hst508_120705_l2)) |
| L20 | RNA Expression: Classification, 2-way Clustering, Regulatory Modules (![This resource may not render correctly in a screen reader.](/images/inacessible.gif)[PDF]({{< baseurl >}}/resources/kho_hst508_lec3))  {{< br >}}  {{< br >}}Integration of Expression and Proteomic Data |
| R6 | Recitation |
| L21 | Dynamics of Biological Networks Metabolic, Regulatory  {{< br >}}  {{< br >}}FBA, Signaling, Regulation of Gene Expression |
| L22 | Medical Lectures - _Prof. Isaac Kohane_  {{< br >}}Two Examples: Phenylketonuria (monogenic) and Diabetes Type 2 (multigenic+)  {{< br >}}  {{< br >}}"Disease" Genes vs. "Susceptibility" Genes  {{< br >}}  {{< br >}}"Environmental" vs. "Developmental" Regulation of Gene Expression |
| L23 | Guest Lecture |
| L24 | Final Presentations